examples/aci_multicast_pool_block/main.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
```

Comment:
move content of this file into the other one. Just one file for both resources. So we can make sure it is clear how to use the two resources separately?

Reply:
Ok do we delete the other example?

Reply:
merge into one example, keep example clearly distinct with different naming

Reply:
done